packages/react-aria-components/src/Form.tsx

```diff
@@ -25,17 +25,17 @@ export interface FormProps extends SharedFormProps, DOMProps {
   validationBehavior?: 'aria' | 'native'
 }
 
-export const FormValidationBehaviorContext = createContext<FormProps['validationBehavior'] | null>(null);
+export const FormContext = createContext<{validationBehavior: FormProps['validationBehavior']} | null>(null);
```

Comment:
for now not adding the rest of the FormProps, not sure if they would be used?

Reply:
i think there was an issue or discussion about this within the last week, but I haven't managed to dig it back up